### PR TITLE
Functioning habit tracker

### PR DESCRIPTION
**Pull request by ChernoAlieu**

### CODE REVIEW

File: chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+								Id INTEGER PRIMARY KEY AUTOINCREMENT,
+								Date TEXT,
+								Quantity INTEGER
+								)";
+			tableCmd.ExecuteNonQuery();
+			connection.Close();
+		}
+
+		GetUserInput();
+
```
Comment (TheCSharpAcademy) [inline chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs:29]: Please remove unnecessary empty line 29. You want to have one line between methods but none between closing curly brackets

File: chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+					break;
+				case "4":
+					Update();
+					break;
+				default:
+					Console.WriteLine("\nInvalid Command. Please type a number from 0 to 4.\n");
+					break;
+			}
+		}
+
```
Comment (TheCSharpAcademy) [inline chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs:73]: Please remove unnecessary empty lines 73/74

File: chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+		using (var connection = new SqliteConnection(connectionString))
+		{
+			connection.Open();
+			var tableCmd = connection.CreateCommand();
+			tableCmd.CommandText = $"INSERT INTO drinking_water(date, quantity) VALUES('{date}',{quantity})";
+
+			tableCmd.ExecuteNonQuery();
+			connection.Close();
+		}
+
```
Comment (TheCSharpAcademy) [inline chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs:94]: Please remove unnecessary empty line 94

File: chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+	}
+
+	internal static string GetDateInput()
+	{
+		Console.WriteLine("\n\nPlease insert the date: (Format: dd-mm-yyyy). Type 0 to return to main menu.");
```
Comment (TheCSharpAcademy) [inline chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs:99]: Looks like you are telling users to insert a year with 4 digits but your datetime parsing method is checking for year with 2 digits 🤓😁

File: chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+
+			if (checkQuery == 0)
+			{
+				Console.WriteLine($"\n\nRecord with Id {recordId} doesn't exist.\n\n");
+				connection.Close();
+				Update();
+			}
+
+			string date = GetDateInput();
+			int quantity = GetNumberInput("\n\nPlease insert number of glasses or other measure of your " +
```
Comment (TheCSharpAcademy) [inline chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs:222]: You can use @ to print your strings verbatim. Check this out: 
image

File: chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+	}
+}
+
+public class DrinkingWater
+{
+	public int Id { get; set; }
+	public DateTime Date { get; set; }
+	public int Quantity { get; set; }
+}
+
```
Comment (TheCSharpAcademy) [inline chernoAlieu.Habit-Tracker/Habit-Tracker/Program.cs:238]: remove empty lines 238,239,240

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

**Comment by ChernoAlieu** [comment]
Thank you Pablo. I made the changes now. Please update my dashboard

**Pull request merged**